lib/bundler/resolver.rb

```diff
@@ -3,12 +3,20 @@ class Resolver
     require "bundler/vendored_molinillo"
 
     class Molinillo::VersionConflict
+      def printable_dep(dep)
+        case dep
+        when DepProxy; dep.to_s.strip
+        when Bundler::Dependency; DepProxy.new(dep, dep.platforms.join(", ")).to_s.strip
+        else; raise("Don't know how to print #{dep.inspect}")
```

Comment:
It seems we only ever have a Bundler::Dep (which is what defines #platforms) in our suite.